getCalspec/rebuild.py

```diff
@@ -67,6 +84,11 @@ def clean_table(df):
     df.set_index("Star_name", inplace=True)
     df.drop(index="[1]", inplace=True)
     df.reset_index(drop=False, inplace=True)
+    # remove _stis from Model column and put it in STIS column
+    for index, row in df.iterrows():
+        if type(row["Model"]) is str and "stis" in row["Model"]:
```

Comment:
I don't know this is necessary because of some pandas weirdness or something, but usually doing type checks should be `isinstance(row["Model"], str)` not with `type` as the `type` way doesn't handle inheritance properly.